flake.nix

```diff
@@ -3,232 +3,234 @@
 
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs";
-    flake-utils.url = "github:numtide/flake-utils";
+
+    systems.url = "github:nix-systems/default";
+
     terranix-examples.url = "github:terranix/terranix-examples";
+
     bats-support = {
       url = "github:bats-core/bats-support";
       flake = false;
     };
+
     bats-assert = {
       url = "github:bats-core/bats-assert";
       flake = false;
     };
   };
 
-  outputs =
-    { self
-    , nixpkgs
-    , flake-utils
-    , terranix-examples
-    , bats-support
-    , bats-assert
-    }:
-    (flake-utils.lib.eachDefaultSystem (system:
-    let pkgs = nixpkgs.legacyPackages.${system};
-    in
-    {
-
-      # nix build
-      packages.terranix = pkgs.callPackage ./default.nix {
-        # as long nix flake is an experimental feature;
-        nix = pkgs.nixUnstable;
-      };
-      # nix build "manpages"
-      packages.manpages = (pkgs.callPackage ./doc/default.nix { }).manPages;
-      packages.default = self.packages.${system}.terranix;
-      # TODO: Legacy attribute, drop soon
-      defaultPackage = self.packages.${system}.default;
-
-      # nix develop
-      devShells.default = pkgs.mkShell {
-        buildInputs =
-          [
-            pkgs.terraform_0_15
-            self.packages.${system}.terranix
-            pkgs.treefmt
-            pkgs.nixpkgs-fmt
-            pkgs.shfmt
-            pkgs.shellcheck
-            pkgs.nodePackages.prettier
-          ];
-      };
-      # TODO: Legacy attribute, drop soon
-      devShell = self.devShells.${system}.default;
-
-      # nix run
-      apps.default = self.apps.${system}.test;
-      # TODO: Legacy attribute, drop soon
-      defaultApp = self.apps.${system}.default;
-      # nix run ".#test"
-      apps.test =
-        let
-          tests = import ./tests/test.nix {
-            inherit nixpkgs;
-            inherit pkgs;
-            inherit (pkgs) lib;
-            terranix = self.packages.${system}.terranix;
+  outputs = inputs @ { self, flake-parts, nixpkgs, ... }:
+
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      imports = [
+        ./flake-module.nix
+      ];
```

Comment:
I  may be wrong but I don't see any option defined in `flake-module.nix` being set by `flake.nix`, is this used somehow?